### PR TITLE
common: remove `SimpleTimeFormat` const for time package layout const `DateTime`

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Also bumps [appveyor windows image](https://www.appveyor.com/docs/windows-images-software/#golang). 🙏 

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ environment:
   PSQL_SKIPSQLCMD: true
   PSQL_TESTDBNAME: gct_dev_ci
   
-stack: go 1.17.8 # this is not actually used on Windows images
+stack: go 1.20.3
```
Comment (thrasher-) [inline .appveyor.yml:32]: Revert as it's not used as suggested, but please add a note to change the PATH at https://github.com/thrasher-corp/gocryptotrader/blob/master/.appveyor.yml#L39 instead

**Review by thrasher- — CHANGES_REQUESTED** [review]
Nice! Looks like there's these ones remaining too:

image

**Review by thrasher- — APPROVED** [review]
tACK, now that the upstream AppVeyor issue has been resolved: https://github.com/appveyor/ci/issues/3884#issuecomment-1621035858

**Comment by gbjk** [comment]
@shazbert Need to bump go to 1.20 in [go.mod](https://github.com/thrasher-corp/gocryptotrader/blob/master/go.mod#L3)

**Pull request merged**